Objective-C/CBLDatabase.mm

```diff
@@ -1044,7 +1044,7 @@ - (void) purgeExpiredDocuments {
         C4Error err;
         NSUInteger nPurged = c4db_purgeExpiredDocs(_c4db, &err);
         CBLLog(Database, @"Purged %lu expired documents", (unsigned long)nPurged);
-        [self scheduleDocumentExpiration: 1.0];
+        [self scheduleDocumentExpiration: 0.0];
```

Comment:
Please add one second back to this for now as we want to have some interval between checking the expired doc to purge. We can discuss an appropriate interval on Friday.